packages/wdio-cucumber-framework/cucumber-framework.d.ts

```diff
@@ -27,10 +27,10 @@ interface StepData {
 
 interface CucumberHookFunctions {
     beforeFeature?(uri: string, feature: CucumberHookObject, scenarios: CucumberHookObject[]): void;
-    beforeScenario?(uri: string, feature: CucumberHookObject, scenario: CucumberHookObject, sourceLocation: SourceLocation): void;
-    beforeStep?(step: StepData, context: any): void;
-    afterStep?(step: StepData, context: any, result: { error?: any, result?: any, passed: boolean, duration: number }): void;
-    afterScenario?(uri: string, feature: CucumberHookObject, scenario: CucumberHookObject, result: CucumberHookResult, sourceLocation: SourceLocation): void;
+    beforeScenario?(uri: string, feature: CucumberHookObject, scenario: CucumberHookObject, sourceLocation: SourceLocation, context?: World): void;
```

Comment:
Do we need to import this type (`World`)?

Reply:
It is defined by Cucumber types and recognized by VSCode without importing anything.  Though still trying to test locally with no success due to "command not found: wdio".

Is this the right procedure?
1) npm run setup-full in webdriverio repo directory
2) run "npm pack" in webdriverio repo directory
3) run 'npm uninstall webdriverio" in project directory
3) run "npm install <path to webdriverio-monorepo.tgz file" in project directory

there is no wdio executable in node_modules\.bin after this.

Reply:
please see our [contributing guidelines](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md#link-changes-to-your-current-project)

Reply:
@christian-bromann , I tried the npm link procedure first but it keeps apparently linking to a global webdrivero, even after I uninstall -g webdriverio, not my modified C:\webderiverio where npm link was run:

 webdriverio@6.1.17
added 139 packages from 187 contributors in 10.23s
C:\osmolyar_qdtools\iris\latest\ManagementPortal\node_modules\webdriverio -> C:\Users\osmolyar\AppData\Roaming\npm\node_modules\webdriverio

Reply:
Also tried just linking packages/wdio-cucumber-framework but when running 'npx wdio ...' it keeps saying 'command not found:  wdio'.  Even after doing 'npm i wdio'.

Reply:
Managed to link @wdio/cucumber-framework but it doesn't seem to be working at all.  (In fact, can't even get context to console.log in before/afterStep where it is supposed to already be supported).  Will close PR for now.